src/logging.cc

```diff
@@ -2627,6 +2654,10 @@ void EnableLogCleaner(const std::chrono::minutes& overdue) {
   log_cleaner.Enable(overdue);
 }
 
+void EnableLogCleaner(std::unordered_map<LogSeverity, std::chrono::minutes>& overdue_per_severity) {
```

Comment:
Exposing the map as a parameter makes the calls verbose and the API usage unnecessarily complicated. Instead use the following signature:
```suggestion
void EnableLogCleaner(LogSeverity severity, const std::chrono::minutes& overdue) {
```